### PR TITLE
run github actions when something is pushed to master

### PR DESCRIPTION
**Pull request by psi29a**

### CODE REVIEW

File: .github/workflows/openmw.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          param(
+            [Parameter(Mandatory=$true)]
+            [String]
+            $FileName
+          )
+          [IO.Path]::GetInvalidFileNameChars() | ForEach-Object {
+            $FileName = $FileName.Replace($_, '_')
+          }
+          return $FileName
+        }
```
Comment (AnyOldName3) [inline .github/workflows/openmw.yml:113]: We only need this for zipping up artefacts, which this isn't doing here.

**Comment by AnyOldName3** [comment]
ytho

**Comment by AnyOldName3** [comment]
Well it's getting a lot further after that change.

**Comment by AnyOldName3** [comment]
Slight problem. No compiler installed on Windows.

**Comment by psi29a** [comment]
Added already in code by way of gitlab :)

**Pull request closed without merging**